src/App.tsx

```diff
@@ -0,0 +1,31 @@
+import { BrowserRouter, Routes, Route, Link } from 'react-router-dom';
+import Home from './pages/home';
+import About from './pages/about';
+import NotFound from './pages/404';
+import Character from './pages/character';
+
+function App() {
+  return (
+    <BrowserRouter>
+      <nav className="border-b-solid m-8 border-b-2 border-b-blue-500 p-2 text-blue-500">
+        <Link to="/" className="m-4 p-1 hover:bg-white">
+          Home
+        </Link>{' '}
+        ●{' '}
+        <Link to="/about" className="m-4 p-1 hover:bg-white">
+          About me
+        </Link>
+      </nav>
+
+      <Routes>
+        <Route path="/" element={<Home />}>
+          <Route path="characters/:id" element={<Character />} />
+        </Route>
+        <Route path="/about" element={<About />} />
+        <Route path="*" element={<NotFound />} />
+      </Routes>
```

Comment:
You should get page into the path. I would recommend you next solution, but you could do it in another way. It's up to you.
```suggestion
      <Routes>
        <Route path="/" element={<Home />} />
        <Route path="/page/:pageId" element={<Home />}>
          <Route path="characters/:id" element={<Character />} />
        </Route>
          <Route path="/page/:pageId" element={<Home />} />
        <Route path="/about" element={<About />} />
        <Route path="*" element={<NotFound />} />
      </Routes>
```

Reply:
ok!